src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  margin: 0 5px;
+  padding: 0 5px;
+  position: relative;
+}
+
+.nav__item:first-child {
+  padding-left: 0;
+  margin-left: 0;
+}
+
+.nav__item:last-child {
+  padding-right: 0;
+  margin-right: 0;
+}
+
+.nav__link {
+  height: 60px;
+  line-height: 60px;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  cursor: pointer;
+}
+
+.is-active1 {
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  text-decoration-line: underline;
+  text-decoration-style: solid;
+  text-decoration-color: #00acdc;
+  text-decoration-thickness: 3px;
+}
```

Comment:
You added some unnecessary hover styles. Here how it looks like on the mockup
![image](https://user-images.githubusercontent.com/60885495/159211170-a4f1d799-14cb-4d9c-97c0-6425c972e770.png)

Reply:
when i was hovering over mock in figma I saw a thick underline below the text so I have done it ....i have changed it now please check
thank you